src/commands/convert/hdf2asff.ts

```diff
@@ -51,15 +52,30 @@ export default class HDF2ASFF extends Command {
     }
 
     if (flags.upload) {
-      const profileInfoFinding: any = converted.pop()
-      const client = new SecurityHubClient({region: flags.region})
+      const profileInfoFinding = converted.pop()
+      convertedSlices = sliceIntoChunks(converted, 100)
+
+      if (flags.insecure) {
+        console.log('WARNING: Using --insecure will make all connections to AWS open to MITM attacks, if possible pass a certificate file with --certificate')
```

Comment:
Do we always use console.log for this type of output?  If so, then leave this as is though it might be worthwhile going through swapping these sorts of messages out for console.error / console.warn.

https://nodejs.org/api/console.html#consolewarndata-args